modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/suggestions/spi/asah/BaseAsahSuggestionsContributor.java

```diff
@@ -190,7 +190,7 @@ protected SuggestionsContributorResults getSuggestionsContributorResults(
 				searchContext, sort, suggestionsContributorConfiguration),
 			"JSONObject/_embedded", "JSONArray/" + path);
 
-		if (jsonArray.length() == 0) {
+		if ((jsonArray == null) || (jsonArray.length() == 0)) {
```

Comment:
Please replace with JsonUtil.isEmpty(JSONArray)

Reply:
Nice It is really better